spec/requests/order_items_spec.rb

```diff
@@ -1,5 +1,12 @@
 describe "OrderItemsRequests", :type => :request do
-  before { disable_tenancy }
+  before do
+    disable_tenancy
+    disable_rbac
+  end
+
+  after do
```

Comment:
Have you thought or tried using `around` hook here?  https://relishapp.com/rspec/rspec-core/v/2-0/docs/hooks/around-hooks

Not sure it buys much but looks a little more concise.